code-challenges.test.js

```diff
@@ -14,44 +16,81 @@
 
 // a) Create a test with an expect statement using the variable provided.
 // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
+describe('shuffledArray', () => {
+  const expected1 = ["yellow", "blue", "pink", "green"]
+  const expected2 = ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+  it('matches even if received contains elements in a different order', () => {
+    const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+    const colors2 = ["chartreuse","indigo","periwinkle","ochre","aquamarine","saffron"]
+    expect(colors1).toEqual(expect.arrayContaining(expected1));
+    expect(colors2).toEqual(expect.arrayContaining(expected2));
+  });
+});
+// b) Create the function that makes the test pass.
+const shuffledArray = (array) => {
+  array.shift();
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = [
-  "chartreuse",
-  "indigo",
-  "periwinkle",
-  "ochre",
-  "aquamarine",
-  "saffron"
-]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+  // Fisher-Yates shuffle algorithm
```

Comment:
Good research on how to shuffle an array, just make sure you understand how it's doing it!